From aaf9e1447db3a778c9f4d46cbb4d3857bf1382c6 Mon Sep 17 00:00:00 2001
From: Hakan Englund <hakan.englund@intel.com>
Date: Mon, 9 Dec 2013 15:49:27 +0100
Subject: REVERTME: Bluetooth frequent auto restart

Version 0x36 of the Wilkins Peak firmwares contain a bug that
leads to frequent auto restarts. By setting HCI_MAX_SIMUL_CMDS
this can be worked around

Issue: APDEV-335
Change-Id: I692aedb9777fd7c1d5faab23b7424af5c3203475
Signed-off-by: Hakan Englund <hakan.englund@intel.com>
---
 include/bt_target.h |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/include/bt_target.h b/include/bt_target.h
index a106773..1f00cd5 100644
--- a/include/bt_target.h
+++ b/include/bt_target.h
@@ -784,7 +784,7 @@ BT_API extern void bte_main_lpm_allow_bt_device_sleep(void);
 
 /* If nonzero, the upper-layer sends at most this number of HCI commands to the lower-layer. */
 #ifndef HCI_MAX_SIMUL_CMDS
-#define HCI_MAX_SIMUL_CMDS          0
+#define HCI_MAX_SIMUL_CMDS          1
 #endif
 
 /* Timeout for receiving response to HCI command */
-- 
1.7.4.1

